label_studio/tests/utils.py

```diff
@@ -80,6 +80,11 @@ def upload_from_string(self, string):
             print(f'String {string} uploaded to bucket {self.bucket_name}')
         def generate_signed_url(self, **kwargs):
             return f'https://storage.googleapis.com/{self.bucket_name}/{self.key}'
+        def download_as_bytes(self):
+            data = f'test_blob_{self.key}'
+            if self.is_json:
+                return json.dumps({'str_field': data, 'int_field': 123, 'dict_field': {'one': 'wow', 'two': 456}})
```

Comment:
I figure it must be working since it's in the tests, but is it alright that this returns a string even though the function name is `download_as_bytes`?

Reply:
@pakelley this was already merged in another PR